DesignPatternAssignment-versionTwo/FactoryPatterAPI/Singleton.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+    namespace FactoryPatterAPI
+    {
+
+        public class Singleton
+        {
+
+            private static Singleton singletonInstance;
+
+            private Singleton()
+            {
+            }
+
+            public static Singleton getSingletonInstance()
+            {
+                if (null == singletonInstance)
+                {
+                    Console.WriteLine("singleton created");
+                    singletonInstance = new Singleton();
+                }
+                return singletonInstance;
+            }
+            public void logger(string message)
```

Comment:
method name should start with capital letter
also logger name doesn't make any sense